Sources/LocalReceiptParsing/ReceiptParser.swift

```diff
@@ -38,9 +38,7 @@ class ReceiptParser {
     }
 
     func parse(from receiptData: Data) throws -> AppleReceipt {
-        let intData = [UInt8](receiptData)
-
-        let asn1Container = try self.containerBuilder.build(fromPayload: ArraySlice(intData))
+        let asn1Container = try self.containerBuilder.build(fromPayload: ArraySlice(receiptData))
```

Comment:
I think this doesn't actually improve performance thanks to Swift's COW behavior, but it cleans up an unnecessary value.

Reply:
I have no idea so I had to ask, does it matter we are creating a ArraySlice on a Data and we were passing a `[UInt8]` before?

Reply:
`Data` is already a `Collection` of `UInt8` values, so this is equivalent to the previous implementation (and we have good test coverage to ensure that).